frontend/src/component/playground/Playground/PlaygroundForm/PlaygroundCodeFieldset/PlaygroundCodeFieldset.tsx

```diff
@@ -213,6 +216,16 @@ export const PlaygroundCodeFieldset: VFC<IPlaygroundCodeFieldsetProps> = ({
         );
     };
 
+    const onContextFieldChange = (event: SelectChangeEvent) => {
```

Comment:
I'd call it changeContextField. General pattern onChange={doSth} instead of onChange={onChangeRepeated}